.gitignore

```diff
@@ -63,7 +63,7 @@ project.fragment.lock.json
 artifacts/
 launchSettings.json
 appsettings.*.json
-
+TestController.cs
```

Comment:
Don't specify some temporary files in gitignore, they shouldn't be attached to git at all